contents/docs/self-host/runbook/postgresql/resize-disk.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Resize disk
```

Comment:
this is basically copy-pasta from kafka, but good to have to link to users directly